rules-tests/Transform/Rector/Class_/RemoveAllowDynamicPropertiesAttributeRector/config/configured_rule.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Transform\Rector\Class_\RemoveAllowDynamicPropertiesAttributeRector;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+    $services->set(RemoveAllowDynamicPropertiesAttributeRector::class)
+        ->call('configure', [[
```

Comment:
please add 1 more test class with no config to fallback to change all classes, even without namespace